modules/sdk/src/main/resources/state_channels_allowance_lists_mainnet.yaml

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
`yaml` does not need `{}` 🤔

Reply:
It looks like empty yaml needs these parenthesis. It fails when I leave just empty file.